cmd/node/main.go

```diff
@@ -642,9 +648,32 @@ func startNode(ctx *cli.Context, log logger.Logger, version string) error {
 
 	log.Debug("NTP average clock offset", "value", syncer.ClockOffset())
 
+	if ctx.IsSet(fullArchive.Name) {
+		generalConfig.StoragePruning.FullArchive = ctx.GlobalBool(fullArchive.Name)
+	}
 	if ctx.IsSet(startInEpoch.Name) {
-		log.Debug("start in epoch is enabled")
 		generalConfig.GeneralSettings.StartInEpochEnabled = ctx.GlobalBool(startInEpoch.Name)
+	}
+	// if FullArchive is enabled, we override the conflicting StoragePruning settings and StartInEpoch as well
```

Comment:
maybe move this code inside the `applyCompatibleConfigs` function?

Reply:
Done.